src/main/java/com/crafter/springboottechiteasy/repositories/TelevisionRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.crafter.springboottechiteasy.repositories;
+
+import com.crafter.springboottechiteasy.models.Television;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+
+@Repository
```

Comment:
Deze annotatie is hier niet nodig, je extend namelijk al van JpaRespoitory en daar wordt dit al geregeld